### ISSUE TITLE
"Combine Getter and Setter" dont work

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Bomb1369/junit-generator-v2#6]
```
If "Combine Getter and Setter" checked or not - we always get 2 tests, one for 
getter and one for setter.
```

Original issue reported on code.google.com by `Russ...@gmail.com` on 17 Jan 2013 at 7:27

**Comment by GoogleCodeExporter** [comment]
```
I will investigate and fix.
```

Original comment by `osborn.j...@gmail.com` on 27 Feb 2013 at 1:59
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Found the problem and committed the fix.
```

Original comment by `osborn.j...@gmail.com` on 27 Feb 2013 at 2:55
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Released fix in build 2.0.10. please test at your convenience.
```

Original comment by `osborn.j...@gmail.com` on 5 Mar 2013 at 1:41
- Changed state: **Fixed**
